packages/xo-web/src/xo-app/backup/new/mirror/index.js

```diff
@@ -68,7 +68,7 @@ const normalize = state => {
   schedules = mapValues(schedules, ({ id, ...schedule }) => schedule)
   settings[''] = {
     ...advancedSettings,
-    reportWhen: reportWhen.value,
+    reportWhen: reportWhen.value ?? reportWhen,
```

Comment:
In which case is `reportWhen` and object containing a `value` property?

Reply:
It is a string when its value comes from the initial state, otherwise it is an object containing `value` and `label` properties, when modified by the user